src/NumField/Selmer.jl

```diff
@@ -281,16 +281,16 @@ function Hecke.is_power(a::FacElem{QQFieldElem, QQField}, p::Int)
   return true
 end
 
-function Hecke.is_power_with_root(a::FacElem{QQFieldElem, QQField}, p::Int)
+function is_power(a::FacElem{QQFieldElem, QQField}, p::Int)
```

Comment:
This signature is now equivalent to the one right above. The above function should probably be removed

Reply:
Removed it